### PR TITLE
LPD-21826 - Add testUI npm script.

### PR DESCRIPTION
**Pull request by ethib137**
https://liferay.atlassian.net/browse/LPD-21826

@izaera we talked about this quite a while ago, but now I'm finally sending a PR for it. Let me know if you want me to change anything. Thanks.

### CODE REVIEW

**Comment by ethib137** [comment]
ci:test:sf

**Comment by ethib137** [comment]
ci:test:relevant

File: modules/test/playwright/package.json [hunk 1]
```diff
@@ -30,7 +30,8 @@
 		"local:setup": "ts-node ./scripts/localSetup.ts",
 		"local:teardown": "ts-node ./scripts/localTeardown.ts",
 		"playwright": "playwright",
-		"test": "playwright test"
+		"test": "playwright test",
+		"testUI": "playwright test --ui"
```
Comment (izaera) [inline modules/test/playwright/package.json:34]: I'd use `test:ui`, as we do for `local:setup` and `local:teardown`... Or change those to 'localSetup` and `localTeardown`...  :shrug:

**Review by izaera — CHANGES_REQUESTED** [review]

File: modules/test/playwright/package.json (hunk 1 above)
Comment (ethib137) [reply modules/test/playwright/package.json:34]: Makes sense... I'll update it. Thanks.

Comment (ethib137) [reply modules/test/playwright/package.json]: @izaera just updated with your suggestion.

**Review by izaera — APPROVED** [review]

**Comment by izaera** [comment]
ci:forward

**Pull request closed without merging**